src/main/java/com/dev/cinema/validation/EmailValidator.java

```diff
@@ -0,0 +1,13 @@
+package com.dev.cinema.validation;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    private static final String EMAIL_REGEXP = "^[A-Za-z0-9+_.-]+@(.+)$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        return email.matches(EMAIL_REGEXP);
```

Comment:
add null check here instead of NotNull annot above field, cause this check may be executed before not null check